src/kinetics/KineticsFactory.cpp

```diff
@@ -10,6 +10,7 @@
 #include "cantera/kinetics/InterfaceKinetics.h"
 #include "cantera/kinetics/EdgeKinetics.h"
 #include "cantera/kinetics/importKinetics.h"
+#include "cantera/kinetics/ReactionFactory.h"
```

Comment:
Why is this needed?

Reply:
it requires `newReaction` ... adding `ReactionFactory.h` to `Kinetics.h` per your suggestion makes this no longer necessary.